wlog/svclog/svc1log/logger_default.go

```diff
@@ -26,39 +57,42 @@ type defaultLogger struct {
 }
 
 func (l *defaultLogger) Debug(msg string, params ...Param) {
-	l.logger.Debug(msg, l.toParams(params)...)
+	l.logger.Debug("", toParams(msg, DebugLevelParam(), params)...)
```

Comment:
Why is the message argument `""` and then passed in `toParams` instead? Guessing it has to do with the base-level logger no longer having a message key, but would like to understand this better since the way the interface is used here isn't intuitive and would like to understand if it's specific to this use case or is a more general issue with the interface after this refactor

Reply:
I think you're right about this being an issue with the interface. The interface is only consumed by the svc1logger, I took a stab at updating the interface to clean this up.

Reply:
Sorry but I think this warrants further discussion -- `LeveledLogger` is one of the main interfaces exported by this package and is a key conceptual component, so changing the interface signature would be a major breaking change and not something we can do flippantly.

Would it break the eventual wrapped logger implementation if the core logger still supported the message but then just translated that to a parameter internally? Or perhaps change the behavioral contract of `LeveledLogger` to state that, if the message is the empty string, then the key/value pair will be omitted entirely instead of outputting a key with an empty value? Happy to sync up directly if that would be more effective as well.